extensions/wikia/EditPageLayout/EditPageLayoutController.class.php

```diff
@@ -228,6 +228,10 @@ public function executeEditPage() {
 			|| $templateType === TemplateClassificationService::TEMPLATE_INFOBOX
 			|| $templateType === TemplateClassificationService::TEMPLATE_CUSTOM_INFOBOX;
 
+		if ( !$wgInfoboxPreviewEnabled || !in_array( strtolower( $wgUser->getGlobalPreference( 'language' ) ), $wgInfoboxPreviewSupportedLanuages ) ) {
```

Comment:
Looks good, but would it make sense to create a method, something like getShowInfoboxPreviewState(), that cotain pretty much everything from line 213 to 232?